azurerm/resource_arm_storage_account.go

```diff
@@ -185,6 +185,15 @@ func resourceArmStorageAccount() *schema.Resource {
 							Elem:     &schema.Schema{Type: schema.TypeString},
 							Set:      schema.HashString,
 						},
+						"default_action": {
+							Type:     schema.TypeString,
+							Optional: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								string(storage.DefaultActionAllow),
+								string(storage.DefaultActionDeny),
+							}, true),
```

Comment:
we're making all new fields in the provider case-sensitive - as such can we make this:

```suggestion
							}, false),
```